.drone.yml

```diff
@@ -7,64 +7,83 @@ platform:
 
 steps:
 - name: build
-  image: rancher/hardened-build-base:v1.14.2-amd64
+  image: rancher/hardened-build-base:v1.16.4b7
   volumes:
-  - name: dockersock
-    path: /var/run
+    - name: docker
+      path: /var/run/docker.sock
   commands:
-  - sleep 20
-  - TAG=${DRONE_TAG} make
+    - TAG=${DRONE_TAG} make
   when:
     event:
-    - tag
+      - push
+      - pull_request
 
-- name: push
-  image: rancher/hardened-build-base:v1.14.2-amd64
-  volumes:
-  - name: dockersock
-    path: /var/run
-  commands:
-  - docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-  - TAG=${DRONE_TAG} make image-push
+- name: docker-build
+  image: plugins/docker
   environment:
-    DOCKER_USERNAME:
-      from_secret: docker_username
-    DOCKER_PASSWORD:
+    TAG: ${DRONE_TAG}
+  settings:
+    purge: false
+    custom_dns: 1.1.1.1
```

Comment:
Why is this necessary?

Reply:
something with drone, we have to do it for rancher, not sure why any more tbh just a thing we add. ill ask eio if it's still necessary.